Store shipping contact info in order metadata (4809)

stracker-phil opened the pull request:
_This PR extends #3463_

### Description

When the new email/phone properties are present in in the `Shipping` object, the values are stored to the DB as order-meta values during order processing.

Two new optional order-meta fields added:
- `_ppcp_paypal_contact_email`
- `_ppcp_paypal_contact_phone`

stracker-phil committed:
🎨 Mark const properties as public

stracker-phil committed:
🗃️ Store custom contact details in order-meta

Narek13 commented on modules/ppcp-wc-gateway/src/Processor/OrderMetaTrait.php:
```diff
@@ -57,11 +58,55 @@ public function add_paypal_meta(
 	}
 
+	/**
+	 * Add order-meta entries with custom contact details
```
```suggestion
	 * Adds order-meta entries with custom contact details.
```

stracker-phil replied:
Nice find! 👍

However, the next PR already changes this comment block so I'll ignore this suggestion 😉

Narek13 reviewed (approved):
Thanks. LGTM

stracker-phil committed:
✅ Update tests to expect the shipping-property

These tests do not confirm the contact module functionality, but ensure existing tests pass, without modifying their logic.

stracker-phil committed:
🔀 Merge branch 'PCP-4808’
